rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1641,6 +1705,102 @@ def assert_binance_import_results(rotki: Rotkehlchen):
+            sequence_index=1,
+            timestamp=ts_sec_to_ms(Timestamp(1640910825)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(500.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Main and Funding Wallet',
+        ), HistoryEvent(
+            identifier=8,
+            event_identifier='BNC_29a2d52a91b9bbb7213d4710ad190d587c4febc197373291289d85ae275ec7a5',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1640912422)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(100.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Spot Account and UM Futures Account',
+        ), HistoryEvent(
+            identifier=9,
+            event_identifier='BNC_29a2d52a91b9bbb7213d4710ad190d587c4febc197373291289d85ae275ec7a5',
+            sequence_index=1,
+            timestamp=ts_sec_to_ms(Timestamp(1640912422)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(100.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Spot Account and UM Futures Account',
+        ), HistoryEvent(
+            identifier=10,
+            event_identifier='BNC_33a197c8698d3de4db57bb5d8a43e48addb28538587f73cd624792036b69a870',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1640912706)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(60.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Spot Account and CM Futures Account',
+        ), HistoryEvent(
+            identifier=11,
+            event_identifier='BNC_33a197c8698d3de4db57bb5d8a43e48addb28538587f73cd624792036b69a870',
+            sequence_index=1,
+            timestamp=ts_sec_to_ms(Timestamp(1640912706)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(60.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Spot Account and CM Futures Account',
+        ), HistoryEvent(
+            identifier=12,
+            event_identifier='BNC_a5e6e8af4008679025ce68ac818ee4ae3f8b9f461d70eeb3d30f20f3c9a8d92f',
```

Comment:
Same for `BNC_a5e6e8af4008679025ce68ac818ee4ae3f8b9f461d70eeb3d30f20f3c9a8d92f`

Also why have so many similar events in a test? If we ever need to modify the tests (which we do often due to logic changes), doing so will become a nightmare!